test/bigtest/tests/export-to-csv-test.js

```diff
@@ -43,14 +45,28 @@ describe('Instances', () => {
     describe('clicking saving instances CQL query button', () => {
       beforeEach(async function () {
         // Timeout to skip enabling animation
-        await new Promise((resolve) => { setTimeout(() => resolve(), 3000); });
+        await wait();
         await inventory.headerDropdownMenu.saveInstancesCQLQueryBtn.click();
       });
 
       it('should hide action items', () => {
         expect(inventory.headerDropdownMenu.saveInstancesCQLQueryBtn.isVisible).to.be.false;
       });
     });
+
+    describe('clicking Save instances UIIDs button with API request set up to fail', () => {
+      beforeEach(async function () {
+        this.server.put('/instance-bulk/ids', {}, 500);
+
+        // Timeout to skip enabling animation
+        await wait();
+        await inventory.headerDropdownMenu.clickSaveInstancesUIIDsBtn();
+      });
+
+      it('should display then error callout', () => {
+        expect(HRIDHandlingInteractor.callout.errorCalloutIsPresent).to.be.true;
```

Comment:
I'm not sure that this interactor should be used here. It might confuse.

Reply:
Agree. Changed to `inventory.callout`.